app/src/main/java/org/mozilla/fenix/home/sessioncontrol/SessionControlController.kt

```diff
@@ -376,6 +378,25 @@ class DefaultSessionControlController(
             metrics.track(Event.PocketTopSiteClicked)
         }
 
+        val baseURL = url.tryGetHostFromUrl()
+        if (SupportUtils.GOOGLE_URL.contains(baseURL)) {
```

Comment:
I think we should just do an exact match of `SupportUtils.GOOGLE_URL` for now since that is the only URL we append the search attribution for at the moment, but this will be useful later. One issue I can see if we did move forward with this is that we would be counting visits to something like https://www.google.com/maps, which will not have the search attribution as intended.